docs/topics_5/web-running-saved-analysis.adoc

```diff
@@ -0,0 +1,16 @@
+
+
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='running_saved_analysis_{context}']
+= Running a saved analysis
+
+You can run the last saved version of a project from the *Analysis results* screen.
+
+.Procedure
+
+. In the web console, click *Analysis results*.
+. Select a *Project*.
+. Click *Run analysis*.
+
+The progress of your analysis is displayed in the *Active analysis* portion of the *Analysis results* screen.
```

Comment:
"portion"? What is that? I don't think that's the right term. More like "section". Check the UI term.

Reply:
I think the best phrasing would be 'The *Active analysis* progress bar displays the progress of your analysis." WDYT?

![image](https://user-images.githubusercontent.com/60698649/101277568-027d1d80-37be-11eb-9c41-20701ea1150b.png)

Reply:
I don't understand this screenshot and its relationship to your comment. Why would you want to say that there is a progress bar when one does not appear? I don't see a progress bar, so I can't comment on the progress bar not starting yet. It might be safer to say that the status of the analysis is displayed.